operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -309,16 +309,33 @@ spec:
 # The QPS value of the Kubernetes client.
 #    ---
 #    qps: 175
+
+##########
+#  ---
+#  cache_config:
```

Comment:
Maintain alphabetical order - move this group up above, under the "auth" group and before "deployment".